packages/client/src/pages/GamePage/GamePage.tsx

```diff
@@ -1,5 +1,7 @@
+import StartScene from '../../game/scenes/StartScene'
+
 function GamePage() {
-  return <h2>Our best game will be here</h2>
+  return <StartScene width={window.innerWidth} height={window.innerHeight} />
```

Comment:
Думаю тут не обязательно width и height передавать через пропсы если они берутся из  window
```suggestion
  return <StartScene  />
```